extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -118,17 +118,34 @@ amp-consent.i-amphtml-consent-ui-iframe-active {
   transform: translate3d(0px, 100vh, 0px) !important;
 }
 
-amp-consent.i-amphtml-consent-ui-iframe-active.i-amphtml-consent-ui-enable-border {
+amp-consent.i-amphtml-consent-ui-enable-border {
   border-top-left-radius: 8px !important;
   border-top-right-radius: 8px !important;
   box-shadow: 0 0 5px 0 rgba(0,0,0, 0.2) !important;
 }
 
+amp-consent.i-amphtml-consent-ui-enable-border-lightbox {
+  border-radius: 8px !important;
+  box-shadow: 0 0 5px 0 rgba(0,0,0, 0.2) !important;
+}
+
 amp-consent.i-amphtml-consent-ui-iframe-active.i-amphtml-consent-ui-in {
   transition: transform 0.5s ease-out !important;
   /* Initial height transform will be set through styles */
 }
 
+amp-consent.i-amphtml-consent-ui-iframe-active-lightbox {
+  box-sizing: border-box;
+  top: 0;
+  right: 0;
+  animation: 0.5s lightbox-appear;
+  animation-fill-mode: forwards;
+  height: var(--lightbox-height);
```

Comment:
question: do we need `!important`? I think `i-amphtml-consent-ui-iframe-active-lightbox` is applied last, but I'm no css expert : )

Reply:
Hmm, I don't think we need it here, since all of these rules seem to be applied.